### PR TITLE
Oguz branch01

### PR DESCRIPTION
**Pull request by oyetkin**
Created initial version of "Running a Notebook"

### CODE REVIEW

**Comment by caitlinadams** [comment]
Hey @oyetkin -- it looks like the build is failing because you haven't included this file in the table of contents, which is what tells Sphinx which notebooks to build. 

The table of contents is stored in the [index.rst](https://github.com/digitalearthafrica/deafrica-training-workshop/blob/oguz_branch01/docs/session_1/index.rst)

You'll need to add:
```
04_running_a_notebook
```
on a new line between the `03_` and `05_` files. Note that you don't need the file extension. 

Once you commit that, the build should hopefully work!

**Comment by caitlinadams** [comment]
Hi @oyetkin , I don't know if you have other content written for this and it's not currently committed, but I'm only currently able to see some of the "Explore a basic notebook" section, and nothing else about running the crop health notebook. I can see you've uploaded all the images, but they're not referenced in this version of the notebook. Could you update the notebook with the rest of the content, and then I can review. Let me know if you're stuck and we can try and troubleshoot.

**Comment by eefaye** [comment]
Hi @oyetkin, the DEAfrica Sandbox has a different folder structure to the DEAustralia one. The current content of the `Running a notebook` page is based on DEAus. For instance, instead of the `Tutorial` folder, it's called `Beginners_guide` in DEAfrica. Unfortunately this also means you're going to have to grab screenshots from the Africa sandbox.

Unless @caitlinadams thinks differently - this might mean we can skip the part on 'Apps', which don't exist in DEAfrica (to my knowledge), and go from the `01_Jupyter_notebooks` intro that you've written to following the `Crop_health` notebook as shown in the video. I am not sure if the video is supposed to be embedded in the page, but even if it is, a written explanation of its contents will be useful to people who prefer that.

This is a bit wordy: in summary

- Keep the `Explore a basic notebook` section and modify for DEAfrica

- Remove the `Using Apps` sections and subsections

- Keep the `Do it yourself` section and modify to follow the `crop_health` video (linked in the Trello card)

- Embed the video `Working with Notebooks - measure crop health`

**Comment by oyetkin** [comment]
@eefaye I have updated the document with Digital Earth Africa specific information as requested.  I'd appreciate it if you could take another look.
